ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -20,6 +20,36 @@
 {%         set protocol_lldp_bypass = dot1x.protocol_lldp_bypass | arista.avd.default('-') %}
 {%         set dynamic_authorization = dot1x.dynamic_authorization | arista.avd.default('-') %}
 | {{ system_auth_control }} | {{ protocol_lldp_bypass }} | {{ dynamic_authorization }} |
+{%         if dot1x.mac_based_authentication or dot1x.radius_av_pair is arista.avd.defined %}
+{%             if dot1x.mac_based_authentication is arista.avd.defined %}
+### 802.1X MAC based authentication
+
+| delay | Hold period |
+| ----- | ----------- |
+{%                 set delay =dot1x.mac_based_authentication.delay | arista.avd.default('-') %}
+{%                 set hold_period =dot1x.mac_based_authentication.hold_period | arista.avd.default('-') %}
+| {{ delay }} | {{ hold_period }} |
+{%             endif %}
+{%             if dot1x.radius_av_pair is arista.avd.defined %}
+
+### 802.1X Radius AV pair
+
+| Service type | framed MTU |
+| ------------ | ---------- |
+{%                 if dot1x.radius_av_pair.service_type is arista.avd.defined(true) %}
+{%                     set radius_av_pair_service = "true" %}
+{%                 else %}
+{%                     set radius_av_pair_service = " - " %}
+{%                 endif %}
+{%                 if dot1x.radius_av_pair.framed_mtu is arista.avd.defined %}
+{%                     set framed_mtu = dot1x.radius_av_pair.framed_mtu %}
+{%                 else %}
+{%                     set framed_mtu = " - " %}
+{%                 endif %}
+| {{ radius_av_pair_service }} | {{ framed_mtu }} |
+
```

Comment:
```suggestion
```

Reply:
This last empty line must be removed to ensure proper spacing in the resulting markdown.

Reply:
I will do it and rerun molecule.